paymentsheet/src/main/java/com/stripe/android/paymentsheet/repositories/CustomerAdapter.kt

```diff
@@ -99,37 +99,37 @@ interface CustomerAdapter {
  */
 @ExperimentalCustomerSheetApi
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-sealed class PersistablePaymentMethodOption(
+sealed class PaymentOption(
```

Comment:
We already have a thing named that 🤔 https://github.com/stripe/stripe-android/blob/524ff374a5354f6c39e9d94133918dc2f21390e9/paymentsheet/src/main/java/com/stripe/android/paymentsheet/model/PaymentOption.kt#L27

This seems pretty confusing.

Reply:
Moved `PaymentOption` inside of `CustomerAdapter` to make it less confusing.